File: FH/Java/SWP4/SWP4_Pritz_UE05/src/swp4/ue05/part1/jdbc/domain/Person.java
package swp4.ue05.part1.jdbc.domain;

public class Person {
    private Long id;
    private String firstName;
    private String lastName;
    private String city;
    private Long zip;
    private String address;
    private String tel;

    public Person(String firstName, String lastName, String city, Long zip, String address, String tel) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.city = city;
        this.zip = zip;
        this.address = address;
        this.tel = tel;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getCity() {
        return city;
    }

    public Long getZip() {
        return zip;
    }

    public String getAddress() {
        return address;
    }

    public String getTel() {
        return tel;
    }

    @Override
    public String toString() {
        return "Person{" +
                "id=" + id +
                ", firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                ", city='" + city + '\'' +
                ", zip=" + zip +
                ", address='" + address + '\'' +
                ", tel='" + tel + '\'' +
                '}';
    }
}

File: FH/Tutorium/SWE/UE05/SWE3_Schwab_Ue05/flugreisen/person.cpp
//
// Created by Anja Schwab on 25.12.22.
//

#include "person.h"

string const &person::get_last_name() {
    return last_name;
}

string const &person::get_first_name() {
    return first_name;
}

string const &person::get_address() {
    return address;
}

string const &person::get_gender() {
    return gender;
}

int const &person::get_age() {
    return age;
}

string const &person::get_credit_card() {
    return credit_card;
}
